epoxy-processortest/src/test/resources/ModelReturningClassType.java

```diff
@@ -0,0 +1,14 @@
+package com.airbnb.epoxy;
+
+public class ModelReturningClassType extends EpoxyModel<Object> {
+  @EpoxyAttribute int value;
```

Comment:
Actually, there is no need for it because we have `testModelWithSuper` which initially was failing in that case. That's how I discovered this case :D